src/components/CardGrid/CardGrid.tsx

```diff
@@ -90,9 +91,14 @@ function CardGrid({
   };
 
   return (
-    <InfiniteScroll pageStart={0} loadMore={loadMore || defaultLoadMore} hasMore={hasLoadMore || defaultHasMore} loader={<InfiniteScrollLoader key="loader" />}>
+    <InfiniteScroll
+      pageStart={0}
+      loadMore={isDefined(loadMore) ? loadMore : defaultLoadMore}
```

Comment:
Is there any difference to this?
```suggestion
      loadMore={loadMore ?? defaultLoadMore}
```

Reply:
It checks both `null` and `undefined`. My idea was to filter `undefined` props, but I don't really have strong preference here.

Reply:
Ooh, I see. OK. I guess it's potentially nice to be able to set it to null, though I'm not sure what the use case would be here specifically.

Reply:
Yeah, that is why I removed it here and used nullish coalescing operator instead.